nixos/tests/all-tests.nix

```
@@ -604,6 +604,7 @@ in {
   networking.scripted = handleTest ./networking/networkd-and-scripted.nix { networkd = false; };
   networking.networkd = handleTest ./networking/networkd-and-scripted.nix { networkd = true; };
   networking.networkmanager = handleTest ./networking/networkmanager.nix {};
+  networking.machineid = handleTest ./networking/machineid.nix {};
```

```suggestion
  networking.machineid = handleTest ./machineid.nix {};
```
if you don't mind I think we should move it outside of "the" networking directory.
The intention behind the directory was to not pollute the top-level with files specific to the three tests above, e.g. `router.nix`. I guess the naming might be a bit of, and can do a pr to rename the folder to something like "networking-configuration-managers"

---

Fixed it it. I moved it outside the folder, and now it just matches the other networking test in the main test folder. Should be fine :)